packages/site/cms-site/src/iframebridge/useBlockPreviewFetch.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+
+import { createFetchInMemoryCache } from "../graphQLFetch/fetchInMemoryCache";
+import { createFetchWithPreviewHeaders, createGraphQLFetch } from "../graphQLFetch/graphQLFetch";
+import { useIFrameBridge } from "./useIFrameBridge";
+
+const cachingFetch = createFetchInMemoryCache(fetch);
+
+export function useBlockPreviewFetch(graphqlApiUrl: string) {
+    const iFrameBridge = useIFrameBridge();
+
+    const clientRef = React.useRef(
```

Comment:
Since it isn't a client anymore (which you vehemently pointed out), we should call this `fetchRef` 😁